docroot/modules/custom/cs_generated_content/generated_content/node/civictheme_page_variations/13.component.automated_list.inc

```diff
@@ -16,3357 +16,142 @@ function _cs_generated_content_create_node_civictheme_page__variations__componen
 
   return [
     [
-      'title' => 'Demo Page, Automated List component, Page content, Unlimited, Light card, Promo Card',
+      'title' => 'Demo Page, Automated List component, Dark single exposed filter, light navigation cards, no title top link with bottom space, with pagination',
```

Comment:
Reduced variants down to 6 variants.

The rationale is: we should test each field option once for spacing, links, filter, multi filters for light and dark and due to their encapsulation of styles / visual changes we should be able to spot any regressions